contract/src/main/java/io/neow3j/contract/NeoURI.java

```diff
@@ -45,8 +45,8 @@ public NeoURI(Neow3j neow3j) {
      * @throws IllegalFormatException if the provided URI has an invalid format.
      */
     public static NeoURI fromURI(String uriString) throws IllegalFormatException {
-        String[] base_and_query = uriString.split("\\?");
-        String[] beginTx = base_and_query[0].split(":");
+        String[] baseAndQuery = uriString.split("\\?");
+        String[] beginTx = baseAndQuery[0].split(":");
 
         if (beginTx.length != 2 || !beginTx[0].equals(NEO_SCHEME)) {
```

Comment:
Same here: can we guarantee that `beginTx` will always have a `0` position?

Reply:
Yes. Same reason as in the above conversation. :-)